sietchtabr/Makefile

```diff
@@ -42,191 +42,195 @@ sietchtabr: \
   /var/www/htdocs/pf/solarized-dark.min.css
 
 /etc/acme-client.conf : acme-client.conf
-	install -o root -g wheel -m 0644 \
+	doas install -o root -g wheel -m 0644 \
```

Comment:
Did all these get `doas`ed so you could run Make as your user so `op` doesn't run as root/admin?

Reply:
Yeah, exactly.